docs/application/web/guides/connectivity/bluetooth.md

```diff
@@ -2,88 +2,180 @@
-
-```
-<tizen:privilege name="http://tizen.org/privilege/application.launch"/>
-<tizen:privilege name="http://tizen.org/privilege/bluetooth"/>
-```
-
 ## Managing the Local Bluetooth Adapter
 
-You can enable or disable the local Bluetooth adapter, and set the device name using the system-provided service through the `ApplicationControl` interface (in [mobile](../../api/latest/device_api/mobile/tizen/application.html#ApplicationControl) and [wearable](../../api/latest/device_api/wearable/tizen/application.html#ApplicationControl) applications).
+You can enable or disable the local Bluetooth adapter, and set the device name using the system-provided service through the `ApplicationControl` interface (in [mobile](../../api/latest/device_api/mobile/tizen/application.html#ApplicationControl), [wearable](../../api/latest/device_api/wearable/tizen/application.html#ApplicationControl) and [tv](../../api/latest/device_api/tv/tizen/application.html#ApplicationControl) applications).
 
 To use the Bluetooth functionality of the device, you must switch the Bluetooth adapter on. The Bluetooth API does not provide a method to enable or disable the Bluetooth adapter of the device directly. When Bluetooth is required, you must request the built-in Settings application on the device to let the user enable or disable Bluetooth.
 
 **Figure: Bluetooth setting screen**
 
 ![Bluetooth setting screen](./media/bluetooth_onoff.png)
 
-1. Retrieve a `BluetoothAdapter` object (in [mobile](../../api/latest/device_api/mobile/tizen/bluetooth.html#BluetoothAdapter) and  [wearable](../../api/latest/device_api/wearable/tizen/bluetooth.html#BluetoothAdapter) applications) with the `getDefaultAdapter()` method and prepare the `ApplicationControl` object (in [mobile](../../api/latest/device_api/mobile/tizen/application.html#ApplicationControl) and  [wearable](../../api/latest/device_api/wearable/tizen/application.html#ApplicationControl) applications) to request the Bluetooth switching operation:
+1. Retrieve a `BluetoothAdapter` object (in [mobile](../../api/latest/device_api/mobile/tizen/bluetooth.html#BluetoothAdapter), [wearable](../../api/latest/device_api/wearable/tizen/bluetooth.html#BluetoothAdapter) and [tv](../../api/latest/device_api/tv/tizen/bluetooth.html#BluetoothAdapter) applications) with the `getDefaultAdapter()` method and prepare the `ApplicationControl` object (in [mobile](../../api/latest/device_api/mobile/tizen/application.html#ApplicationControl), [wearable](../../api/latest/device_api/wearable/tizen/application.html#ApplicationControl) and [tv](../../api/latest/device_api/tv/tizen/application.html#ApplicationControl) applications) to request the Bluetooth switching operation:
```

Comment:
Please add a serial comma before tv

Reply:
I have added serial commas before "tv" in the whole file.